### PR TITLE
Update bibliography.bib

### PR DESCRIPTION
**Pull request by YihanLiu4023**

### CODE REVIEW

**Comment by YihanLiu4023** [comment]
@TuomasBorman Hi Tuomas, I just created a new PR to see if this is correct and no conflicts. Please let me know if this workds! Thank you!

**Review by antagomir — APPROVED** [review]

**Comment by antagomir** [comment]
Looks good. I will wait for @TuomasBorman to merge.

File: inst/assets/bibliography.bib [hunk 1]
```diff
@@ -3,18 +3,6 @@
...
-@Article{Willis2019,
-  author =	 {Amy D. Willis},
-  title =	 {Rarefaction, Alpha Diversity, and Statistics},
-  journal =	 {Frontiers in Microbiology},
-  year =	 {2019},
-  volume =	 10,
-  doi =		 {10.3389/fmicb.2019.02407}
-}
-
-
```
Comment (TuomasBorman) [inline inst/assets/bibliography.bib]: Why this is removed?

File: inst/assets/bibliography.bib [hunk 2]
```diff
@@ -1941,235 +1969,217 @@ @article{Haidich2010
...
-    volume = {9},
-    number = {2},
-    pages = {e00354-23},
-    doi = {10.1128/msphere.00354-23},
-    year = {2024}
-}
-
-@article{Schloss2024rarefaction2,
-    author = {Patrick D. Schloss},
-    title = {Waste not, want not: revisiting the analysis that called into question the practice of rarefaction},
```
Comment (TuomasBorman) [inline inst/assets/bibliography.bib]: Why these are removed?

**Review by TuomasBorman — CHANGES_REQUESTED** [review]
Looks good!

There are couple of references that are completely removed. That is ok, if they are not used in book, just wanted to check

File: inst/assets/bibliography.bib (hunk 2 above)
Comment (antagomir) [reply inst/assets/bibliography.bib]: I would keep the refs that are there. We might like to cite them later when finalizing the text.

**Review by TuomasBorman — APPROVED** [review]

**Pull request merged**